config.yaml

```diff
@@ -20,6 +20,9 @@ params:
   font:
     name: "Lato"
     sizes: [400,900]
+  plausible:
+    dataDomain: scipy.org
+    javaScript: "https://analytics.scientific-python.org/js/plausible.js"
```

Comment:
You don't need to specify javaScript here, since that is the default.  But it won't hurt either.

Reply:
Let's remove it then since either way it's linked to the theme.